### PR TITLE
DRAFT: G2 Eletroshock grenade

### PR DESCRIPTION
**Pull request by PhantomEpicness**
# About the pull request
Adds the G2 electroshock grenade, "Sonic Electric ball breaker" as mentioned in the movies or SEBB for short. It's purpose is to be a tactical option for the marines, as it deals slowdown instead of stun/damage. (unless you stand on top of it)

Only meaningful damage is to marines and especially synths, since a 1.2 Gigavolt charge is probably not healthy for bioelectrics.

Can be used as a mine as well. Uses an effect similar to boiler neuro to have finer control over balance and to prevent stacking. (still wip)

Made a PR so maintainers can take a look at it. Still WIP

# Explain why it's good for the game
More marine utility weapons and content, we can probably lower the amount of HEDPs slightly to balance it out aswell.

+ lore accurate af, as mentioned in the movie and the 

# Testing Photographs and Procedure

<details>
<summary>Screenshots & Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

# Changelog

:cl: TotalEpicness5, Triiodine, monkeyfist
add: Adds the G2 electroshock grenade
/:cl:

### CODE REVIEW

**Comment by cm13-github** [comment]
This pull request has conflicts, please resolve those before we can evaluate the pull request.

File: code/game/objects/items/explosives/grenades/marines.dm [hunk 1]
```diff
@@ -713,6 +713,110 @@
...
+		if(prob(25))
+			var/datum/effect_system/spark_spread/spark = new /datum/effect_system/spark_spread
+			spark.set_up(2, 1, turf)
+			spark.start()
+	for(var/mob/living/Mob in oview(range,src))
+		var/mob_range = get_dist(src,Mob)
+		to_chat(world,mob_range)  /// REMOVETHIS REMOVETHIS
+		var/dam_factor = mob_range * dam_range_mult
+		to_chat(world,dam_factor)  /// REMOVETHIS REMOVETHIS
+		var/damage_applied = damage - dam_factor  //divides damage
```
Comment (Zonespace27) [inline code/game/objects/items/explosives/grenades/marines.dm:774]: reminder to remove

File: code/game/objects/items/explosives/grenades/marines.dm [hunk 2]
```diff
@@ -713,6 +713,110 @@
...
+
+/obj/item/explosive/grenade/sebb/primed/New()
+	activate()
+
+/obj/effect/overlay/sebb
+	name = "Danger"
+	icon = 'icons/effects/sebb.dmi'
+	icon_state = "sebb_explode" // testing
+	layer = ABOVE_LIGHTING_PLANE
+	var/time_to_live = 1 SECONDS
```
Comment (Zonespace27) [inline code/game/objects/items/explosives/grenades/marines.dm:810]: unique vars at the bottom

File: code/game/objects/items/explosives/mine.dm [hunk 3]
```diff
@@ -59,6 +59,22 @@
 		return TRUE
 
 
+/obj/item/explosive/mine/sebb
+	name = "\improper G2 Electroshock grenade"
+	icon_state = "grenade_sebb_planted"
+	desc = "A G2 electroshock grenade planted as a landmine"
```
Comment (Zonespace27) [inline code/game/objects/items/explosives/mine.dm:65]:
```suggestion
	desc = "A G2 electroshock grenade planted as a landmine."
```

File: code/game/objects/items/explosives/mine.dm [hunk 4]
```diff
@@ -59,6 +59,22 @@
...
+	map_deployed = TRUE
+
+/obj/item/explosive/mine/sebb/disarm()
+	. = ..()
+	new /obj/item/explosive/grenade/sebb(src)
+	qdel(src)
+
+/obj/item/explosive/mine/sebb/prime()
+	set waitfor = 0
+	new /obj/item/explosive/grenade/sebb/primed(src.loc)
```
Comment (Zonespace27) [inline code/game/objects/items/explosives/mine.dm:75]: not necessarily guaranteed to be a turf

File: code/game/objects/items/explosives/mine.dm [hunk 5]
```diff
@@ -59,6 +59,22 @@
...
+	if(!QDELETED(src))
+		disarm()
```
Comment (Zonespace27) [inline code/game/objects/items/explosives/mine.dm:77]: what's this check for?

**Review by Zonespace27 — CHANGES_REQUESTED** [review]
Non-balance

**Pull request closed without merging**